nemesis-generator/configuration_generator/__main__.py

```diff
@@ -1,23 +1,28 @@
 import argparse
 import secrets
+import time
 
 import yaml
 from symbolchain.CryptoTypes import Hash256, PrivateKey
 from symbolchain.facade.NemFacade import NemFacade
+from symbolchain.nem.Network import Network
 from zenlog import log
 
 MICROXEM_PER_XEM = 1000000
 
 
 class NemesisConfigurationGenerator:
-	def __init__(self, network):
-		self.facade = NemFacade(network)
+	def __init__(self, network_name, network_identifier):
+		self.epoch_time = int(time.time_ns() / 1000000000)
+		self.facade = NemFacade(network_name)
```

Comment:
is there a reason you're not creating the facade around the network object?

Reply:
you mean overriding the network object in the facade?  yea, I was thinking about doing this but since I am using the network for only creating the address then just decided to keep it separate.  
Is that a better approach?

Reply:
i meant `self.facade = NemFacade(self.network)`

Reply:
ok.  will take a look.  didnt realize the ``NemFacade`` takes a network object in the constructor.

Reply:
ah, that's right ... it's NEM 😅